docker-compose-app.yml

```diff
@@ -6,6 +6,26 @@ services:
     build: web
     ports:
       - 80:3000
+    networks:
+      - default
+    deploy:
+      replicas: 3
+      update_config:
+        parallelism: 1
+        delay: 5s
+      restart_policy:
+        condition: on-failure
+      placement:
+        constraints:
+          - node.role == worker
```

Comment:
Does swarm not make this constraint by default? i.e. if I omit it, could I end up with an app container on my manager node? Second question: is there any disadvantage to that happening?